### PR TITLE
Added new documant

### PR DESCRIPTION
**Pull request by AllenBEden**
Added Contributors Section and more documant in this "index.html" file....

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -785,7 +785,181 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
...
+							<img src="https://avatars2.githubusercontent.com/u/1583873?s=460&v=4">
+							</div>
+							<a href="https://github.com/mariobehling" target="_blank">
+							<h5><span>Mario Behling</span></div></h5>
+							</a>
+						</div>
+					</div>
+				<div class="col-xs-6 col-sm-4 col-md-3 col-lg-2">
+					<div class="card"><div class="avatar">
+						<img src="https://avatars3.githubusercontent.com/u/14032427?s=460&v=4">
```
Comment (abishekvashok) [inline index.html:811]: Don't add contributors statically, use the github api.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Won't it be a huge task adding contributttors manually ? :)

File: index.html (hunk 1 above)
Comment (jigyasa-grover) [reply index.html:811]: Closing wrt PR #54.

**Comment by jigyasa-grover** [comment]
Closing wrt PR #54.

**Pull request closed without merging**